src/bin/collapse-dtrace.rs

```diff
@@ -0,0 +1,70 @@
+use env_logger::Env;
+use std::io;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+use inferno::collapse::dtrace::{Folder, Options};
+use inferno::collapse::Collapse;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "inferno-collapse-perf",
+    author = "",
+    after_help = "\
+[1] perf script must emit both PID and TIDs for these to work; eg, Linux < 4.1:
+        perf script -f comm,pid,tid,cpu,time,event,ip,sym,dso,trace
+    for Linux >= 4.1:
+        perf script -F comm,pid,tid,cpu,time,event,ip,sym,dso,trace
+    If you save this output add --header on Linux >= 3.14 to include perf info."
+)]
+struct Opt {
+    /// include raw addresses where symbols can't be found
+    #[structopt(long = "addrs")]
+    include_addrs: bool,
```

Comment:
I think this is called `includeoffset` [upstream](https://github.com/brendangregg/FlameGraph/blob/1b1c6deede9c33c5134c920bdb7a44cc5528e9a7/stackcollapse.pl#L54), but given that it's not a command-line argument there, we should feel free to call it whatever we want to here. So let's stick with `addrs` for consistency with `collapse-perf` :)

Reply:
👍 removed it as it's not passable to upstream.